src/reflect.ts

```diff
@@ -13,9 +13,12 @@ export type Reflect =
   | ({ tag: 'symbol' } & Runtype<symbol>)
   | ({ tag: 'literal'; value: LiteralBase } & Runtype<LiteralBase>)
   | ({ tag: 'array'; element: Reflect; isReadonly: boolean } & Runtype<ReadonlyArray<unknown>>)
-  | ({ tag: 'record'; fields: { [_: string]: Reflect }; isReadonly: boolean } & Runtype<
-      { readonly [_ in string]: unknown }
-    >)
+  | ({
+      tag: 'record';
+      fields: { [_: string]: Reflect };
+      isPartial: boolean;
+      isReadonly: boolean;
+    } & Runtype<{ readonly [_ in string]: unknown }>)
   | ({ tag: 'partial'; fields: { [_: string]: Reflect } } & Runtype<{ [_ in string]?: unknown }>)
```

Comment:
The `partial` reflect variant should be removed since it's been subsumed by `record` now, right?

Reply:
Yep, missed  that, my bad.